### ISSUE TITLE
Schedules don't show up when upgrading from 0.5.1 to 0.6.1

### ISSUE THREAD

**Issue by iancward** [matsukaz/hubot-schedule#15]
Greetings,
I'm using hubot with hubot-slack version 4.5.4. I attempted to upgrade by bot's dependency on hubot-schedule from 0.5.1 to 0.6.1, but during testing, none of the scheduled events show up with `@hubot schedule list`.
I downgraded hubot-schedule back to 0.5.1 and things seem to work again.

This is what (some) of the data in the `hubot:storage` key in redis looks like:
```json
{
	"_private": {
		"hubot_schedule": {
			"1564": ["0 0 * * *", {
					"room": "C1A2BCD3E",
					"id": "U0FGHI12J",
					"team_id": "T0ABCD1EF",
					"name": "ian.ward"
				}, " test message!"]
		}
	}
}
```

**Comment by iancward** [comment]
I think this issue is because the [RTM Data store](https://github.com/matsukaz/hubot-schedule/blob/master/src/scripts/schedule.coffee#L248)  has been [deprecated](https://github.com/slackapi/hubot-slack/pull/465).

**Comment by matsukaz** [comment]
Thank you for creating this issue.
I'll check if this problem happens on my environment too.

**Comment by matsukaz** [comment]
@iancward 
I've fixed the bug and released as v0.6.2.
Sorry for my late response X(
I would be really glad if you could try it and check if this issue has been solved or not.
